builtin/types/plasma_cash/plasma_cash.proto

```diff
@@ -45,6 +45,12 @@ message PlasmaBlock {
     bytes proof = 6;
 }
 
+message ReplayProtection {
+    uint64 nonce = 1 [jstype = JS_STRING]; // Nonce for replay protection
+    bytes signature = 2; // Signature to recover sender from
```

Comment:
The signature is the only thing that actually needs to be sent over, so just add it to the `PlasmaTxRequest`.